packages/util-dev/test/TestLogger.test.ts

```diff
@@ -87,7 +87,7 @@ describe('TestLogger', () => {
     });
 
     describe('TL_BREAK', () => {
-      const loggedData = Object.fromEntries(Array.from({ length: 10 }, (v, i) => [i.toString(), `v${i}`]));
+      const loggedData = Object.fromEntries(Array.from({ length: 10 }, (_v, i) => [i.toString(), `v${i}`]));
```

Comment:
Why `_v` here?

Reply:
Because it is unused. Without the underscore `eslint` raises an error (or warning...)

Reply:
Oh. I see. Didn't notice it's not used. I've seen the following conventions until now.
```
(_, i) => [i.toString(), `v${i}`];
// or
(, i) => [i.toString(), `v${i}`];

Reply:
I'm using that one `(_, i) => [i.toString(), `v${i}`];`

Reply:
b6849e8a6 should solve this